client/components/pages/HomePage/HomePage.js

```diff
@@ -1,32 +1,74 @@
-import React, { useEffect } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { push } from 'connected-react-router';
-import R from 'ramda';
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { push } from "connected-react-router";
+import R from "ramda";
 
-import Section from 'react-bulma-companion/lib/Section';
-import Container from 'react-bulma-companion/lib/Container';
-import Title from 'react-bulma-companion/lib/Title';
+import Section from "react-bulma-companion/lib/Section";
+import Container from "react-bulma-companion/lib/Container";
+import Title from "react-bulma-companion/lib/Title";
+import InputCircle from "../../atoms/InputCircle";
+import ArrowInput from "../../atoms/ArrowInput";
+import EventBoard from "../../molecules/EventsBoard/EventsBoard";
+import Button from "../../atoms/Button/Button";
+import PageLayout from "../../organisms/PageLayout";
+import Gallery from "../../molecules/Gallery/Gallery";
 
-export default function HomePage() {
+const HomePage = () => {
+  HomePage.treeImg = false;
+  HomePage.title = "חורשה 80";
+  const [inputLeft, setInputLeft] = useState("0.8");
```

Comment:
Why is this in a state?
To prepare for taking the value from the redux?

Reply:
yes!